Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -186,16 +186,25 @@ public extension Sequence {
         return (Array(matching), Array(nonMatching))
     }
 
+    /// SwifterSwift: Return a map grouped on a keypath value.
+    ///
+    /// - Parameter by: Key path to group by.  Must be Hashable.
+    /// - Returns: Dictionary of arrays using the keypath value as the key.
+    func grouped<T: Hashable>(by keyPath: KeyPath<Element, T>) -> [T: [Element]] {
+        // https://twitter.com/johnsundell/status/1106886417756704768?lang=en
+        return .init(grouping: self, by: { $0[keyPath: keyPath] })
```

Comment:
I used to like `.init(...)`, but not anymore...
```suggestion
        return Dictionary(grouping: self, by: { $0[keyPath: keyPath] })
```